sotodlib/preprocess/processes.py

```diff
@@ -330,50 +330,66 @@ class Noise(_Preprocess):
     Saves the results into the "noise" field of proc_aman. 
 
     Can run data selection of a "max_noise" value. 
-    
+
+    Example config block::
+
+     - name: "noise"
+       calc:
+         low_f: 5
+         high_f: 10
+       save: True
+       select:
+         max_noise: 2000
+
     .. autofunction:: sotodlib.tod_ops.fft_ops.calc_wn
```

Comment:
This function is only called if fit = False, if True, fit_noise_model is called.

Reply:
Want me to add the `fit_noise_model` autofunction here too then?

Reply:
After talking about this at lunch, I'm planning on linking to where both of these functions are documented instead. Will get to that later this afternoon.

Reply:
Alright, linked to this in [da22eb0](https://github.com/simonsobs/sotodlib/pull/749/commits/da22eb0502d83efea8e7dcbbd718e5c5b77478bf). I had to add an automodule block to the `tod_ops` docs page.